### PR TITLE
Updated newuser doc for phub

### PR DESCRIPTION
**Pull request by sjletcher**
This will just need the link for Vincent's more detailed doc added

### CODE REVIEW

File: docs/newuser.md [hunk 1]
```diff
@@ -1,29 +1,41 @@
...
-This document will go over the flow for getting access to our various services
+This document will go over the flow for getting access to our various services if you are finding difficulties with the specifics of the tasks outlined in this document please see the more detailed [Developer Docs]()
 
-## Publicly Accessible
+## Office 365 Account
+[Office 365](https://portal.office.com), these are provided by emailing DigitalCapabilitiesPMO@homeoffice.gsi.gov.uk, this is not managed by us. Please note that for Single Sign On (SSO) which is using your Office 365 account to log onto other services pop-ups will need to be enabled on your computer.
+
+## VPN
+The VPN is used to be able to get through to our multiple AWS environments as well as other data centers. The profiles can be downloaded from [authd](https://authd.digital.homeoffice.gov.uk) which you can access using your office 365 SSO. The following services can be only accessed when connected to the Dev VPN profile.
+
```
Comment (vinn946) [inline docs/newuser.md]: I'm thinking we should also note that access to London stuff (acp drone, platform hub etc.) requires the London VPNs through here `https://access-acp.digital.homeoffice.gov.uk` 
What do you think?

File: docs/newuser.md [hunk 2]
```diff
@@ -1,29 +1,41 @@
 # Flow for new users
-This document will go over the flow for getting access to our various services
+This document will go over the flow for getting access to our various services if you are finding difficulties with the specifics of the tasks outlined in this document please see the more detailed [Developer Docs]()
 
-## Publicly Accessible
+## Office 365 Account
+[Office 365](https://portal.office.com), these are provided by emailing DigitalCapabilitiesPMO@homeoffice.gsi.gov.uk, this is not managed by us. Please note that for Single Sign On (SSO) which is using your Office 365 account to log onto other services pop-ups will need to be enabled on your computer.
```
Comment (jon-shanks) [inline docs/newuser.md]: We should probably remove this content and remove it from the git history. Perhaps make it more high-level i.e. SSO is managed as part of an on-boarding process, which will be managed by your project manager / delivery manager to the office 365 tema.

Comment (jits) [reply docs/newuser.md]: +1, we tend to get a lot of confusion about this. So what @jon-shanks said makes it clearer that the ACP team would not be able to help with getting O365 access.

File: docs/newuser.md [hunk 3]
```diff
@@ -1,29 +1,41 @@
...
+
+## VPN
+The VPN is used to be able to get through to our multiple AWS environments as well as other data centers. The profiles can be downloaded from [authd](https://authd.digital.homeoffice.gov.uk) which you can access using your office 365 SSO. The following services can be only accessed when connected to the Dev VPN profile.
+
+## Platform Hub
+
+## Externally Accessible
```
Comment (jon-shanks) [inline docs/newuser.md]: We should probably mention that these are for open source projects only

File: docs/newuser.md (hunk 1 above)
Comment (jits) [reply docs/newuser.md]: Given these docs will be solely for ACP London (I assume?) we should just have the London VPN link here.

File: docs/newuser.md [hunk 4]
```diff
@@ -1,29 +1,41 @@
...
+This document will go over the flow for getting access to our various services if you are finding difficulties with the specifics of the tasks outlined in this document please see the more detailed [Developer Docs]()
 
-## Publicly Accessible
+## Office 365 Account
+[Office 365](https://portal.office.com), these are provided by emailing DigitalCapabilitiesPMO@homeoffice.gsi.gov.uk, this is not managed by us. Please note that for Single Sign On (SSO) which is using your Office 365 account to log onto other services pop-ups will need to be enabled on your computer.
+
+## VPN
+The VPN is used to be able to get through to our multiple AWS environments as well as other data centers. The profiles can be downloaded from [authd](https://authd.digital.homeoffice.gov.uk) which you can access using your office 365 SSO. The following services can be only accessed when connected to the Dev VPN profile.
+
+## Platform Hub
```
Comment (jits) [inline docs/newuser.md]: When we were discussing, I was thinking the Platform Hub should be it's own section, separate to the ones below. Technically, all the services below aren't part of (or provided by) the hub. Just that in terms of _ordering_, getting access to the hub first would help with onboarding on to certain services (like GitHub).

File: docs/newuser.md [hunk 5]
```diff
@@ -1,29 +1,41 @@
...
+
+## Externally Accessible
 These are all services that you can access through normal internet access with personal accounts.
 ### GitHub
-[Github](https://github.com/UKHomeOffice) to access our repositories you must have your personal Github invited to the UK Home Office organisation on Github, someone with access can request this for you.
+[Github](https://github.com/UKHomeOffice) to access our repositories you must have your personal Github invited to the UK Home Office organisation on Github, someone with access can request this for you. Please note that you will have to have your full name as your name on github along with 2 Factor Authentication. If you don't have a personal account on Github then you will need to create one.
```
Comment (jits) [inline docs/newuser.md]: Onboarding on to the GitHub org can now be done via the hub – we probably don't want support requests about this anymore?

File: docs/newuser.md [hunk 6]
```diff
@@ -1,29 +1,41 @@
...
+
 ### Quay
-[Quay](https://quay.io) You can access this with your github account after requesting access to our Quay Organisation with your personal account. Requests should be raised on the [bau board](https://github.com/UKHomeOffice/application-container-platform-bau/issues)
+[Quay](https://quay.io) You can access this with your github account after requesting access to our Quay Organisation with your personal account. Requests can be made on the [Platform Hub](https://hub.acp.homeoffice.gov.uk/help/support/requests/overview)
```
Comment (jits) [inline docs/newuser.md]: `github` -> `GitHub`

Comment (jits) [reply docs/newuser.md]: Also, is there a direct link to the support request form you could put here?

File: docs/newuser.md [hunk 7]
```diff
@@ -1,29 +1,41 @@
...
 
-## Office 365 Account
-[Office 365](https://portal.office.com), these are provided by emailing DigitalCapabilitiesPMO@homeoffice.gsi.gov.uk, this is not managed by us.
+## Internally Accessible
 
 ### Artifactory
-[Artifactory](https://artifactory.digital.homeoffice.gov.uk.). Sign in with office 365 button. The docker repository in artifactory is accessed by the docker.digital.homeoffice.gov.uk address.
+[Artifactory](https://artifactory.digital.homeoffice.gov.uk.). Requests can be made on the [Platform Hub](https://hub.acp.homeoffice.gov.uk/help/support/requests/overview) Sign in with Office 365 button. The docker repository in artifactory is accessed by the docker.digital.homeoffice.gov.uk address.
```
Comment (jits) [inline docs/newuser.md]: Likewise, with the direct link to the support request form.

File: docs/newuser.md [hunk 8]
```diff
@@ -1,29 +1,41 @@
...
+
 ### Gitlab
-[GitLab](https://gitlab.digital.homeoffice.gov.uk) To get access to this you will need to first login via office 365 by clicking the office 365 login button and and then request for your account to be unblocked.
-## VPN
-The VPN is used to be able to get through to our multiple AWS environments as well as other data centers. The profiles can be downloaded from [authd](https://authd.digital.homeoffice.gov.uk) which you can access using your office 365 SSO. The following services can be only accessed when connected to the Dev VPN profile.
+[GitLab](https://gitlab.digital.homeoffice.gov.uk) To get access to this you will need to first login via office 365 by clicking the office 365 login button requests can be made on the [Platform Hub](https://hub.acp.homeoffice.gov.uk/help/support/requests/overview)
```
Comment (jits) [inline docs/newuser.md]: Need some sort of punctuation for between `office 365 login button` and `requests can be made` (unless I'm misreading it).

Also, it's not clear what kind of support request can/should be made to gain access? (Same point here about using a direct link to a support request form, to make it a tad more useful)

File: docs/newuser.md [hunk 9]
```diff
@@ -1,29 +1,41 @@
...
-[Kibana](https://kibana.ops.digital.homeoffice.gov.uk) uses office365 sigin.
+[Kibana](https://kibana.ops.digital.homeoffice.gov.uk) uses Office 365 sign in.
+
 ### Sysdig
-[Sysdig](https://sysdig.digital.homeoffice.gov.uk). You need to request an account on our [bau board](https://github.com/UKHomeOffice/application-container-platform-bau/issues) if you want access.
+[Sysdig](https://sysdig.digital.homeoffice.gov.uk). Requests can be made on the [Platform Hub](https://hub.acp.homeoffice.gov.uk/help/support/requests/overview) 
+
 ## AWS
-Access to AWS is managed by the [Hoddat-iam](https://gitlab.digital.homeoffice.gov.uk/Devops/hoddat-iam) repo.
+Access to AWS is also managed by the Platform Hub. Requests can be made on the [Platform Hub](https://hub.acp.homeoffice.gov.uk/help/support/requests/overview)
```
Comment (jits) [inline docs/newuser.md]: Access to AWS is not quite "managed by the Platform Hub" – I think it's a case of using the "general support" request form to ask the ACP team to grant access?

**Comment by jon-shanks** [comment]
LGTM

**Pull request merged**